src/views/collection-view/index.tsx

```diff
@@ -59,12 +59,11 @@ function CollectionView({
 			sidecarSlot={null}
 		>
 			<CollectionMeta
+				collection={collection}
```

Comment:
Here, and in the other similar view files that use `CollectionMeta`, we switch things up to use the newly required `collection` prop and drop the `cta` and `numTutorials` props.